okhttp/src/test/java/okhttp3/ConnectionSpecTest.java

```diff
@@ -181,6 +181,21 @@
         .build();
   }
 
+  @Test public void tls_recomputesCiphersAndVersions() throws Exception {
```

Comment:
Sounds like this is what we expect instead of using a cache?

Reply:
I don't want this test. It's not a behavior we guarantee either way. Just don't want PRs that intend to add Kotlin idioms also adding caches.

Reply:
sure! removed the test :)